src/main/java/frc/robot/subsystems/drive/Drivetrain.java

```diff
@@ -325,12 +325,12 @@ private void move(Vector spdVec, double rot, boolean rateLimit) {
 
     spdCommanded.setX(spdVec.x());
     spdCommanded.setY(spdVec.y());
-    // Adjust input based on max speed
-    spdCommanded.mult(DriveConfig.kMaxSpeedMetersPerSecond);
 
     if (rateLimit) {
       limitDirectionSlewRate(spdCommanded);
       m_currentRotationRadians = m_rotLimiter.calculate(rot);
+      // Adjust input based on max speed
+      spdCommanded.mult(DriveConfig.kMaxSpeedMetersPerSecond);
```

Comment:
This is wrong. If you're not limiting the slew rate, the multiplication never happens. The multiplications happens after the if statement.